.circleci/config.yml

```
@@ -0,0 +1,33 @@
+version: 2.1
+
+orbs:
+  aws-ecr: circleci/aws-ecr@7.3.0
+
+jobs:
+  build-and-push-image:
+    parameters:
+      postgres-version:
+        type: string
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 19.03.13
+      - aws-ecr/build-and-push-image:
+          account-url: AWS_ECR_ACCOUNT_URL
+          aws-access-key-id: AWS_ACCESS_KEY_ID
+          aws-secret-access-key: AWS_SECRET_ACCESS_KEY
+          region: AWS_ECR_DEFAULT_REGION
+          repo: ${CIRCLE_PROJECT_REPONAME}
+          extra-build-args: "--rm --build-arg POSTGRES_VERSION=<<parameters.postgres-version>>"
+          tag: <<parameters.postgres-version>>
+
+workflows:
+  build:
+    jobs:
+      - build-and-push-image:
+          context: aws-ecr
+          matrix:
+            parameters:
+              postgres-version: ["9.5", "9.6", "10.19", "11.14", "12.9", "13.5", "14.1"]
```

This is the list of versions we have deployed

regular postgres: 9.6.22
aurora postgres (clinical/reporting etc): 12.4
aurora postgres (notifications): 10.14
```suggestion
              postgres-version: ["9.6", "10.14", "12.4"]
```